requirements.txt

```diff
@@ -3,3 +3,4 @@ django-model-utils>=4.0
 jsonfield>=3.1.0,<4.0.0
 cryptography>=2.4.0,<2.9.0
 pyopenssl>=17.5.0,<20.0.0
+openwisp_utils>=0.4.0,<0.5.0
```

Comment:
I added these because the UI of `django_x509` was not like the other `openwisp module's`. It was still having plain django's UI. I wanted to look like the other modules. But since it was not requested, i will remove it